File: CS50X/pset6/mario/more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for i in range(height):
    for space in range(height - i - 1):
        print(" ", end = "")
    row = (2*i) + 3
    for block in range(row):
        if block == (row - 1) / 2:
            print("  ", end = "")
        else:
            print("#", end = "")
    print()
        

File: CS50X/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main (void)

{
    int i;
    do
    {
        i=get_int("Height: ");
    }
    while(i<1 || i>8);
    
   int x,y,z,p;
    for(y=0; y<i; y++)
    {
        for(z=0; z <i-y-1 ;z++)
        {
            printf(" ");
        }
        for(x=0; x<y+1; x++)
        {
            printf("#");
        }
        printf("  ");
        for(p=0; p<y+1; p++)
        {
            printf("#");
        }
    printf("\n");
    }
}
    